inst/extdata/OSD/A/ACKWATER.json

```diff
@@ -96,9 +96,9 @@
         "dry_hue": "NA",
         "dry_value": "NA",
         "dry_chroma": "NA",
-        "moist_hue": "NA",
-        "moist_value": "NA",
-        "moist_chroma": "NA",
+        "moist_hue": "lOYR",
```

Comment:
Many previously NA colors are now at least identified, hues may need cleanup before writing out to JSON, but I think we should at least make a first pass to try and clean these up. If we ultimately need to do a more radical fix for import into NASIS OSD structure then we can do that later.